### ISSUE TITLE
wis2box-ui needs a11y improvements

### ISSUE THREAD

**Issue by C-Loftus** [wmo-im/wis2box#846]
_Noting this to myself for future dev work; I can get to this, just might be a bit_

the wis2box-ui needs some labeling to be accessible for assistive technology users

- waiting on data in the test server at  http://136.156.130.194/ but I believe the map of observations and associated pop up window for data charts do not take keyboard focus correctly and need to be labelled 
- on the front page maps are labeled as `image` and not something descriptive
- the logo on the front page is just labeled as `logo` and not something more descriptive
- error popups should be labelled with `aria-live` to assert to a screen reader that it is a notification
    - it does not take keyboard focus and is not tab navigable 
- the `cluster` button is not keyboard triggerable with the spacebar / enter key
